addons/website/static/src/snippets/s_features_grid/000.scss

```diff
@@ -1,13 +1,11 @@
 
 .s_features_grid {
     .s_features_grid_content {
-        overflow: hidden;
         p {
             margin-bottom: 0;
         }
     }
     .s_features_grid_icon {
-        float: left;
         margin-right: $grid-gutter-width/2;
```

Comment:
We may add a `gap-2`class on the wrapper and remove this CSS rule as well -> thus remove the entire file ;)